src/mol-plugin-ui/spec.ts

```diff
@@ -33,6 +34,7 @@ interface PluginUISpec extends PluginSpec {
         disableDragOverlay?: boolean,
         selectionTools?: {
             controls?: React.ComponentClass | React.FC,
+            granularityOptions?: Set<Loci.Granularity>,
```

Comment:
Please make this a plain array
```suggestion
            granularityOptions?: Loci.Granularity[],
```